pinecone/config/config.py

```diff
@@ -46,10 +46,10 @@ def build(
         if not host:
             raise PineconeConfigurationError("You haven't specified a host.")
 
-        openapi_config = (
-            openapi_config
-            or kwargs.pop("openapi_config", None)
```

Comment:
`or kwargs.pop("openapi_config", None)` wasn't doing anything, since `openapi_config` is a named param and the key should never appear in `kwargs`.